TaskManagement/registration/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from registration.views import Dashboard
+
+
+urlpatterns = [
+
+    path('', Dashboard.as_view(), name='dashboard'),
```

Comment:
instead of making a view to render a page use template view in url search for it. you will need to only pass template name here.